clients/withManifest.ts

```diff
@@ -92,13 +94,29 @@ const fetchWithProps = async (
     );
   }
   const headers = new Headers(init?.headers);
+  const fetcher = init?.fetcher ?? fetch;
+  const multipart = init?.multipart ?? false;
 
   headers.set("accept", `application/json, text/event-stream`);
-  headers.set("content-type", "application/json");
 
-  const response = await (init?.fetcher ?? fetch)(url, {
+  if (!headers.has("content-type")) {
+    headers.set("content-type", "application/json");
+  }
+
+  if (multipart) {
+    // Let the browser handle the content-type header, with a webkit generated boundary
+    headers.delete("content-type");
+  }
```

Comment:
it seems that an error should be thrown here. If there is set a content-type, but also you want multipart.. maybe you did something wrong

Reply:
Agree, is not much intuitive deleting a content-type header, it should not be added or let the error occur

Reply:
Which one do you guys think its better:
- do a console.warn that your content-type header is being overriden since you're using multipart: true
- throw an error if you try to set both

Reply:
@mcandeia @matheusgr